src/Faker/Provider/ru_RU/Payment.php

```diff
@@ -0,0 +1,811 @@
+<?php
+
+namespace Faker\Provider\ru_RU;
+
+class Payment extends \Faker\Provider\Payment
+{
+    /**
+     * @see list of Russian banks (2015-04-04), source: http://www.banki.ru/banks/
+     * @example "cat *.html | grep 'b-cb-list__name' | iconv --f windows-1251 --t utf-8 | grep -o '>.*<' | \
+     * sed -r 's/&mdash;//' | sed -r 's/[\<\>]//g' | sed -r "s/(^|$)/'/g" | sed -r 's/$/,/' | sed -r 's/\&(laquo|raquo);/"/g' | \
+     * sed -r 's/\s+/ /g'"
+     */
+    protected static $banks = array(
```

Comment:
You absolutly right! But i've no enough information about bank odes in russia. It seems that there are multiple bank registers for banks. So it's hard to get bank id.

That's why i didn't set any ids for banks. Maybe i'll add bank codes in future.